diff --git a/Libraries/LogBox/UI/LogBoxNotificationMessage.js b/Libraries/LogBox/UI/LogBoxNotificationMessage.js
index ae5c182ae8380dd717d31706897598eef493cee4..5600065d4a581caddfbaf50ff489154659e57a0b 100644
--- a/Libraries/LogBox/UI/LogBoxNotificationMessage.js
+++ b/Libraries/LogBox/UI/LogBoxNotificationMessage.js
@@ -44,17 +44,17 @@ const styles = StyleSheet.create({
     flexGrow: 1,
     flexShrink: 1,
     flexBasis: 'auto',
-    borderLeftColor: LogBoxStyle.getTextColor(0.2),
+    borderLeftColor: LogBoxStyle.getBackgroundColor(0.2),
     borderLeftWidth: 1,
     paddingLeft: 8,
   },
   text: {
-    color: LogBoxStyle.getTextColor(1),
+    color: LogBoxStyle.getBackgroundColor(1),
     flex: 1,
     fontSize: 14,
     lineHeight: 22,
   },
   substitutionText: {
-    color: LogBoxStyle.getTextColor(0.6),
+    color: LogBoxStyle.getBackgroundColor(0.6),
   },
 });
